sample/src/main/java/co/nimblehq/sample/ui/login/LoginViewModel.kt

```diff
@@ -20,19 +21,18 @@ class LoginViewModel : ViewModel() {
     val authenService = SurveyApi.instance.buildService<AuthService>()
 
     fun login() {
-
         // Hardcoding the credential information because we dont have UI for this one
         val request = LoginRequest(grantType = "password", email = "dev@nimblehq.co", "12345678")
         viewModelScope.launch {
             withContext(Dispatchers.IO) {
                 try {
                     val result = authenService.loginEmail(request)
-                    SurveyApi.instance.setTokenKey(result.data.attributes.accessToken)
-                    _loginResult.postValue(Result.success("Hello"))
-
+                    SurveyApi.instance.setTokenKey(result.accessToken ?: "")
+                    SurveyApi.instance.setTokenType(result.tokenType ?: "")
```

Comment:
OPTIONAL: `?: ""` == `.orEmpty()`. The code looks nicer with the Kotlin extension 🤞

Reply:
@hoangnguyen92dn Good to know 👌